CAM/SetTableGamesCameraThisUpdate.md

```diff
@@ -0,0 +1,40 @@
+---
+ns: CAM
+aliases: ["_0x79C0E43EB9B944E2"]
+---
+## SET_TABLE_GAMES_CAMERA_THIS_UPDATE
+
+```c
+// 0x79C0E43EB9B944E2
+BOOL SET_TABLE_GAMES_CAMERA_THIS_UPDATE(Hash hash);
+```
+
+Override to table game camerawork for current frame only.
+
+```lua
+eTableGames = {
```

Comment:
enums are typically done in c/cpp fashion

```c
enum eTableGames {
  LuckyWheel = 5891389,
  Slots = 518572876,
  Roulette = 71681063,
  Blackjack = -2124244681,
  ThreeCard = -1938411241,
}
```

Reply:
@AvarianKnight I had written that, but fix it when the reviewer pointed it out.